docs/guides/chip_tool_guide.md

```diff
@@ -913,6 +951,8 @@ will be required by the second Matter admin to join Matter device to its fabric.
 [1663675289.149445][56387:56392] CHIP:CTL: SetupQRCode: [MT:4CT91AFN00YHEE7E300]
 ```
 
+##### Option B: Commissioning to a new fabric
```

Comment:
I don't understand the "Option A"/"Option B" thing.  These are not options you choose between.  These are steps, that need to be done in order.

Reply:
Will rephrase. @kkasperczyk-no , can you have a look at the updated text?